e2e/src/components/Footer.ts

```diff
@@ -0,0 +1,58 @@
+import { Page, Locator, expect } from '@playwright/test';
+import BaseComponent from '../base/BaseComponent';
+
+export default class Footer extends BaseComponent {
+    private readonly footerLocator = this.page.locator('footer#footer');
+    private readonly copyrightLocator = (this.page.locator('.copyright'));
+
+    constructor(page: Page) {
+        super(page);
+    }
+
+    static FooterConstants = {
+        COPYRIGHT_TEXT: '© 2025 Vadym Nastoiashchyi',
+    };
```

Comment:
you may use this hardcoded text on test scenario level. No need to implement it so deep